app/models/registration_payment.rb

```diff
@@ -29,6 +29,37 @@ def payment_status
     end
   end
 
+  def to_v2_json(helpers, refunds: false)
+    payment_provider = CompetitionPaymentIntegration::INTEGRATION_RECORD_TYPES.invert[self.receipt_type]
```

Comment:
why not use key(self.receipt_type) instead of invert?

Reply:
(a) because `invert` was used before (see diff) and (b) because I only knew `key?` existed but I didn't know about `key` :D